backend/homework/5/src/main/java/shrishti/example/question1/Message.java

```diff
@@ -0,0 +1,29 @@
+package shrishti.example.question1;
+
+public class Message {
+    public static  void main(String[] args){
```

Comment:
The class can be cleaner by breaking it into sub-functions for senders. receivers, etc